patches/0037-Restore-legacy-bucket.patch

```diff
@@ -5,319 +5,508 @@ Subject: [PATCH] Restore legacy bucket
 
 
 diff --git a/go.mod b/go.mod
-index 0d8ccda18d..85b3de0453 100644
+index 5b8c85d7d3..a145fa7f16 100644
```

Comment:
This patch diff is hard to review. Couldn't spot anything wrong though

Reply:
We should probably take upstream/go.mod churn out of patches and try to see if we can accept the unmodified upstream go.mod.

Reply:
https://github.com/pulumi/pulumi-aws/issues/4465